src/components/RadioButton/examples/interactive.jsx

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+import React from 'react';
+
+import RadioButton from '../RadioButton';
+
+export default React.createClass({
+	getInitialState() {
+		return {
+			isSelected: false
+		};
+	},
+
+	handleSelected(isSelected) {
+		this.setState(_.assign({}, this.state, {
+			isSelected
+		}));
+	},
+
+	render() {
+		return (
+			<ul>
+				<li>
+					<label>Enabled</label>
```

Comment:
Once we have a pattern for putting labels with our control components, we should refactor the examples to use that pattern.